Sources/SwiftLexicalLookup/Scopes/ScopeImplementations.swift

```diff
@@ -512,13 +500,19 @@ import SwiftSyntax
 @_spi(Experimental) extension CatchClauseSyntax: ScopeSyntax {
   /// Implicit `error` when there are no catch items.
   @_spi(Experimental) public var defaultIntroducedNames: [LookupName] {
+    var containsExpressionSyntax = false
+
     let extractedNames = catchItems.flatMap { item in
       guard let pattern = item.pattern else { return [LookupName]() }
 
+      if !containsExpressionSyntax && pattern.is(ExpressionPatternSyntax.self) {
+        containsExpressionSyntax = true
+      }
+
```

Comment:
This is because we suppress the implicit `error` when there is some kind of pattern? Please leave such a comment with a code example to remind our future selves.